web/src/app/projects/repository/repository.component.scss

```diff
@@ -123,29 +128,32 @@
 
       &__time-container-bottom {
         width: 250px;
-        display: flex;
         justify-content: flex-start;
+        div {
+          display: flex;
+        }
       }
 
       &__time-container-top {
         width: 250px;
-        display: flex;
         justify-content: flex-start;
+        div {
+          display:flex;
+        }
       }
 
       @media (max-width: $breakpoint-md) {
         &__time-container-top {
           width: 120px;
           display: flex;
-          position: absolute;
           font-size: 9pt;
           right: 0;
           top: 0;
         }
 
         &__time-container-bottom {
           width: 120px;
-          display: flex;
+          display: block;
```

Comment:
I think this is causing the layout to break, I will put screenshots in the PR comments

Reply:
No this one is not causing the issue. there is another css `bottom: -5px` which is causing . I will remove it